mw-webapp/src/component/tooltip/positionTooltip/PositionTooltip.ts

```diff
@@ -0,0 +1,6 @@
+export const enum PositionTooltip {
+  TOP = "tooltip_top",
```

Comment:
1.let's avoid BEM, let's use cameCase for className, it will be easier to work with them in tsx 
2. let 's use the same key and value for enums if possible

Reply:
1. ok
2. fixed